modules/salt/pages/config-modules.adoc

```diff
@@ -0,0 +1,39 @@
+[[config-modules]]
+= Configuration Modules
+
+[IMPORTANT]
+====
+This feature is a technology preview.
+====
+
+You can use Salt execution and state modules to configure your {productname} Server.
+These configuration modules can be used directly or using SLS files.
+They are are especially useful if you have multiple {productname} Servers, for example in Hub installations, but they are also useful for smaller installations.
+
+For more information about using Hub, see xref:large-deployments:multi-server.adoc[].
+
+You can use configuration modules to configure:
+
+* Organizations
+* Users
+* User permissions
+* System groups
+
+More configuration modules are planned to be released in the future.
+
+* For more information about Salt execution modules and how to use them, see https://docs.saltstack.com/en/latest/topics/tutorials/modules.html.
+* For more information about Salt state modules, see https://docs.saltstack.com/en/latest/topics/tutorials/starting_states.html.
+
+
+
+== Install Configuration Modules
+
+The Salt modules are available in the [path]``uyuni-config-modules`` package.
```

Comment:
Again this sentence does not sound correct: [many](https://docs.saltstack.com/en/latest/ref/states/all/index.html) [many](https://docs.saltstack.com/en/latest/ref/modules/all/index.html) Salt modules exist and most of them are not part of that package. The sentence is supposed to refer only to the Uyuni configuration states.

Reply:
I think we have a `[package]` tag.